Executive.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+
+Humberto Dominguez
+Professor Scott Roueche
+CSE 687- Object Oriented Design
+Syracuse University
+Phase 1 - Executive cpp
+4/20/2023
+
+This class was created in part of creating the MapReduce code for Phase 1 for
+CSE 687. The Executive class's primary purpose is to gather all data related to directories.
+
+Since this is the cpp portion of the class, this will define how and where the the member and methods will be
+used.
+*/
+
+//libraries that will be used in this class
+#include "Executive.h"
+
+//default constructor 
+Executive::Executive() {}
+
+//default destructor 
+Executive::~Executive() {}
+
+//mathod that will obtain the directories and test to verify they exist
+void Executive::obtainingDirectories() {

```

Comment:
If executive class is supposed to be the 'main' class, then we may need to call this the main method with argc and argv inputs. we can review when we integrate more

Reply:
Does it have to be done through argc and argv? It just says command line. As long as they compile the code, and the Executive class is within main and calls workflow it'll accomplish the same

Reply:
Yes that is true - i am reviewing the instructions again and you are right, it may be easier to do it without argc and argv like you are in this commit. good catch!